### ISSUE TITLE
Add Support for Simple Solid Axel

### ISSUE THREAD

**Issue by MichaelMurguia** [digicreatures/rigacar#61]
David thank you so much for this _FREE_ add-on.  it is very much appreciated. I am a novice rigger and I was using the addon for a solid axel 4wd vehicle and wondering how I would modify the rig to support a simple solid axel between the front and rear wheels. 

I may be able to do this easily by customizing the bones in the rig, but as a novice, i thought it might be helpful to have a "simple axel" checkbox or something. 

This is not complex IFS or anything - basically a pole. 

Thanks for listening! 

M
![image](https://user-images.githubusercontent.com/43971566/152062283-533d525d-6ff8-4253-a7dd-18826ff9e18b.png)

**Comment by spoonless** [comment]
Yep, It could be a good idea. But unfortunately, I have no time to improve this add-on.
For your model, you can try to display the last layer of the armature and you will find two bones: MCH-Axis.Ft for the front and MCH-Axis.Bk for the back.
You can try to parent your axles to those bones. It will probably not be 100% correct but it may do the job. 

![axis bones](https://user-images.githubusercontent.com/1173511/152705955-1dd8eb03-67b0-44e5-b123-e55b43dcb818.png)
